### ISSUE TITLE
Add in a "processing" status to reviews and reviewSummations

### ISSUE THREAD

**Issue by cwdcwd** [topcoder-platform/submissions-api#164]
add another status to the reviews and reviewSummations objects with a value of "processing"

**Comment by callmekatootie** [comment]
@cwdcwd reviewSummation does not have any `status` field. There's an `isFinal` flag on it though...

Do you mean to say that it will now contain a status field too?

**Comment by dushyantb** [comment]
@callmekatootie yes, like the `review` will be having status for `reviewSummation` as well,
also, right now actual status values are in source js file; that should be moved to config; so that in future in case we need to add any new status, only need to modify config file

**Comment by callmekatootie** [comment]
F2F launched - http://www.topcoder.com/challenge-details/30114675/?type=develop

**Comment by callmekatootie** [comment]
Additions:

- review summation can also have status of queued
- in case status is queued or processing, `aggregateScore` needs to be null

**Comment by callmekatootie** [comment]
Pull Request ready - https://github.com/topcoder-platform/submissions-api/pull/167
